### DISCUSSION TITLE
Why does the UI look different?

### DISCUSSION THREAD

**Discussion by harry0703** [rustdesk/rustdesk#5941] [General]
I downloaded the `rustdesk-1.2.2-x86_64.dmg` and built the `source code`, but I noticed that the user interface (UI) appears to be different.
For instance, in the source code, the "Options" is implemented as a popup menu, whereas in the pre-built version, it is displayed as a tabbed window.

Thank you.

![Snipaste_2023-10-07_18-12-54](https://github.com/rustdesk/rustdesk/assets/4928832/f400da1d-bd09-45a3-8c8a-2ae9c3d6df9c)

**Comment by SergeyMy** [comment]
Desktop versions use **Sciter** or **Flutter** for GUI!
